sale_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# © 2019 ForgeFlow S.L.
+# - Jordi Ballester Alomar
+# © 2019 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Operating Unit in Sales",
+    "version": "14.0.1.0.0",
+    "summary": "An operating unit (OU) is an organizational entity part of a "
+    "company",
+    "author": "ForgeFlow, "
+    "Serpent Consulting Services Pvt. Ltd.,"
+    "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/operating-unit",
+    "category": "Sales Management",
+    "depends": ["sale", "account_operating_unit", "sales_team_operating_unit"],
```

Comment:
change dependecy "sale" to "sale_management" since it's not installing the module

Reply:
Functional review installing the sales module manually is working ok